build.gradle

```diff
@@ -35,7 +36,8 @@ def versions = [
     sonarPitest        : '0.5',
     springBoot         : '2.2.4.RELEASE',
     springHystrix      : '2.1.1.RELEASE',
-    springfoxSwagger   : '2.9.2'
+    springfoxSwagger   : '2.9.2',
+    pact_version       : '3.5.24'
```

Comment:
Just a question.  There are later versions of Pact now (v4).  Since it is a major release version does this mean more dev work is required to move to it?

Reply:
I tried this upgrade but there is more dev work involved here and so reverted the attempt.